core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -265,6 +266,22 @@ public boolean isNull(RexNode node) {
         return anyNull(rexCall.getOperands());
       }
       return false;
+    case DIVIDE: {
```

Comment:
why does the nullability of the result depend on the argument being a literal?

Reply:
If I understand Julian's comment correctly, division by 0 should throw an exception, so IS NULL(10/0) may not be simplified. However, IS NULL(10/1) may be simplified to false.

Reply:
The JavaDoc on this function says nothing about exceptions, so neither of them should be considered NULL.

Reply:
Maybe we shouldn't bother with zero literals in this particular API. If we have `null / X` or `X / null` the result should be null and it doesn't really matter if X is zero. This is my interpretation of the SQL standard and probably what the function used to do before the changes.

Reply:
Indeed, I've updated the PR. We still need to change the policy to CUSTOM, as it is checked by RexSimplify.